common/slides.mk

```diff
@@ -0,0 +1,15 @@
+RVMD = reveal-md
+
+.PHONY: all html clean
+
+all: html
+
+html: _site
+
+_site: slides.md ../../common/favicon.ico
+	$(RVMD) $< --static $@
+	cp ../../common/favicon.ico $@
+
```

Comment:
I see you've removed the recipe for `slides.pdf`. Is it because of the issues on your part that you mentioned, or do you want it removed completely?

Reply:
I wasn't able to easily create a correct PDF file. I thought this is rather irrelevant since HTML is the output we want. The only benefit of PDF is to print out slides. In which case students could do that from the browser.

I'm open to having a PDF generation rule, if we can find an easy way of doing it inside the Makefile.

I [saw](https://github.com/webpro/reveal-md#2-using-docker--decktape) there is the option of using [DeckTape](https://github.com/astefanutti/decktape), but I did't spend time investigating that.